packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -267,27 +280,41 @@ export class Viewer {
       this.load(this.src);
     }
 
+    if (this.keyboardControls) {
+      this.keyStateInteractionHandler = new KeyStateInteractionHandler();
+      this.registerInteractionHandler(this.keyStateInteractionHandler);
+    }
+
     if (this.cameraControls) {
       // default to pointer events if allowed by browser.
       if (window.PointerEvent != null) {
+        const tapInteractionHandler = new TapInteractionHandler(
+          'pointerdown',
+          'pointerup',
+          'pointermove',
+          () => this.getConfig()
+        );
+
         this.registerInteractionHandler(new PointerInteractionHandler());
         this.registerInteractionHandler(new MultiPointerInteractionHandler());
-        this.registerInteractionHandler(
-          new TapInteractionHandler(
-            'pointerdown',
-            'pointerup',
-            'pointermove',
-            () => this.getConfig()
-          )
+        this.registerInteractionHandler(tapInteractionHandler);
+        this.registerTapKeyInteraction(
+          new FlyToPartKeyInteraction(this.stream, () => this.getConfig())
         );
       } else {
+        const tapInteractionHandler = new TapInteractionHandler(
+          'mousedown',
+          'mouseup',
+          'mousemove',
+          () => this.getConfig()
+        );
+
         // fallback to touch events and mouse events as a default
         this.registerInteractionHandler(new MouseInteractionHandler());
         this.registerInteractionHandler(new TouchInteractionHandler());
-        this.registerInteractionHandler(
-          new TapInteractionHandler('mousedown', 'mouseup', 'mousemove', () =>
-            this.getConfig()
-          )
+        this.registerInteractionHandler(tapInteractionHandler);
+        this.registerTapKeyInteraction(
+          new FlyToPartKeyInteraction(this.stream, () => this.getConfig())
```

Comment:
does this need to be duplicated in both the `if & else`? 

I think this could live inside the `if (this.cameraControls) {...}` conditional.

Reply:
Yep, you're right -- this is leftover from some changes I made. I'll update this